backend/services/auth.js

```diff
@@ -1,6 +1,26 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+import usersService from '../services/users.js';
+
 export default class AuthService {
-  static login() {
-    return 'You have successfully logged in!';
+  static login(email, password) {
+    const user = usersService.getUserByEmail(email);
+
+    if (user && bcrypt.compareSync(password, user.password)) {
+      const payload = {
+        id: user.id,
+        email: user.email,
+        role: user.role,
+      };
+
+      const token = jwt.sign(payload, process.env.JWT_SECRET, {
+        expiresIn: process.env.JWT_EXPIRES_IN,
+      });
+
+      return token;
+    }
+    return null;
```

Comment:
usually some error is thrown when login is unsuccessful

Reply:
If login is unsuccessful this login function returns null. In this case auth controller instead of receving token receves null and
sends response with 401 status code and message about invalid crediantials.
![image](https://user-images.githubusercontent.com/55427679/158573873-2faa2b65-843c-4c36-8d70-3b7c398ab114.png)